executor.go

```diff
@@ -706,7 +719,7 @@ func localExecute(pipeline *zjson.Pipeline, executionId int64, executionUuid str
 				var jsonObj map[string]interface{}
 				err := json.Unmarshal([]byte(message), &jsonObj)
 				if err != nil {
-					fmt.Printf("Failed to log: %s", message)
+					log.Printf("Failed to log: %s", message)
```

Comment:
Needs to stay `fmt` bc it's inside the logger, already, and this can cause infinite loop of logging

Reply:
So when should it be fmt and when should it be log?